articles/api-auth/tutorials/password-grant.md

```diff
@@ -59,6 +59,30 @@ The response from `/oauth/token` (if successful) contains an `access_token`, for
 If the client needs the user's claims you can include the scopes `openid profile` to the `scope` value in the POST to the token endpoint. If the audience uses RS256 as the signing algorithm, the `access_token` will now also include `/userinfo` as a valid audience. You can now send the `access_token` to `https://${account.namespace}/userinfo` to retrieve the user's claims.
 :::
 
+### Realm Support
+
+A extension grant that offers similar functionality with the **Resource Owner Password Grant**, including the ability to indicate a specific realm, is the `password-realm`.
+
+Realms allow you to keep separate user directories and specify which one to use to the token endpoint.
+
+To use this variation you will have to change the following request parameters:
+* Set the `grant_type` to `password-realm`.
```

Comment:
`grant_type` should have a value of `http://auth0.com/oauth/grant-type/password-realm`